src/serializers/encoder/v11/encode_graph_entities.c

```diff
@@ -358,9 +355,10 @@ void RdbSaveEdges_v11
 			M = Graph_GetRelationMatrix(gc->g, r, false);
 			info = RG_MatrixTupleIter_reuse(iter, M);
 			ASSERT(info == GrB_SUCCESS);
-			info = RG_MatrixTupleIter_next_UINT64(iter, &src, &dest, &edgeID, &depleted);
-			ASSERT(info == GrB_SUCCESS);
+			info = RG_MatrixTupleIter_next_UINT64(iter, &src, &dest, &edgeID);
 		}
+		
+		ASSERT(info == GrB_SUCCESS);
```

Comment:
I think it's possible to break out of the loop with the following conditions:
```
1. info = GxB_EXHAUSTED
2. r >= relation_count
```
Which can cause this assertion to fail